consensus/istanbul/core/preprepare.go

```diff
@@ -27,8 +27,8 @@ import (
 func (c *core) sendPreprepare(request *istanbul.Request, roundChangeCertificate istanbul.RoundChangeCertificate) {
 	logger := c.newLogger("func", "sendPreprepare")
 
-	// If I'm the proposer and I have the same sequence with the proposal
-	if c.current.Sequence().Cmp(request.Proposal.Number()) == 0 && c.isProposer() {
+	// If I'm validating, the proposer, and I have the same sequence with the proposal
+	if c.current.Sequence().Cmp(request.Proposal.Number()) == 0 && c.isProposer() && c.backend.IsValidating() {
```

Comment:
why the `isValidating()`? shouldn't core be "stop" if we are not validating?

Reply:
Again, old check.